### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RobbertVH**
Generalizations

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,27 @@
 <a name="readme-top"></a>
-# Microsoft Teams Integration / Dataminer Bot
+# Chat Integration / Dataminer Bot
```
Comment (JordyGit) [inline README.md:2]:
```suggestion
# Chat Integration
```

File: README.md [hunk 2]
```diff
@@ -1,31 +1,27 @@
...
+* The [CloudGateway DxM](https://docs.dataminer.services/user-guide/Cloud_Platform/CloudAdminApp/Managing_cloud-connected_nodes.html) on your Dataminer System must be atleast version 2.9.0.
+* The Dataminer App must be allowed in your [Teams](https://docs.microsoft.com/en-us/microsoftteams/manage-apps).
 
 ### Installing
 
-* Deploy the [DMAPP]() to your Dataminer system. Double-click to install, this will add the automation scripts to your system.
+* Deploy the [DMAPP]() to your Dataminer system. Double-click to install, this will add the automation scripts in the Automation module.
 
-* Access [DataMiner](https://docs.dataminer.services/user-guide/Getting_started/Accessing_DataMiner/Accessing_DataMiner.html) and open the Automation module (via apps > Automation).
+* Access the [DataMiner](https://docs.dataminer.services/user-guide/Getting_started/Accessing_DataMiner/Accessing_DataMiner.html) System and open the Automation module (via apps > Automation).
```
Comment (JordyGit) [inline README.md:24]:
```suggestion
* [Access the DataMiner System](https://docs.dataminer.services/user-guide/Getting_started/Accessing_DataMiner/Accessing_DataMiner.html) and open the Automation module (via apps > Automation).
```

File: README.md [hunk 3]
```diff
@@ -1,31 +1,27 @@
...
-### Limitations
-
-This chat integration is only supported by cloud-connected DataMiner systems.
-
 ### Prerequisites
 
-* [DataMiner](https://community.dataminer.services/dataminer-client-requirements/) installed.
-* The [CloudGateway DxM](https://docs.dataminer.services/user-guide/Cloud_Platform/CloudAdminApp/Managing_cloud-connected_nodes.html) must be atleast version 2.9.0.
-* The Dataminer App must be allowed in the Team.
+* The DataMiner System that integrates with chats must be connected to the DataMiner Cloud Platform ([DCP](https://docs.dataminer.services/user-guide/Cloud_Platform/CloudAdminApp/Managing_cloud-connected_nodes.html)).
```
Comment (JordyGit) [inline README.md:16]:
```suggestion
* The DataMiner System must be cloud-connected to the DataMiner Cloud Platform ([DCP](https://docs.dataminer.services/user-guide/Cloud_Platform/CloudAdminApp/Managing_cloud-connected_nodes.html)).
```

File: README.md [hunk 4]
```diff
@@ -1,31 +1,27 @@
...
+# Chat Integration / Dataminer Bot
 
-This guideline is about how to integrate Dataminer into Microsoft Teams with automation scripts.
+This guideline is about how to integrate Dataminer into business communication platforms with automation scripts.
 
 ## Overview
 
-There are two main parts in this guideline:
- * Adding the automation scripts & memory files to your Dataminer system.
+There are two main parts to the Chat Integration for Teams:
```
Comment (JordyGit) [inline README.md:8]: Generalize and add sub topic for Teams

**Review by JordyGit — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -1,56 +1,78 @@
...
 
-There are two main parts in this guideline:
- * Adding the automation scripts & memory files to your Dataminer system.
- * Granting admin consent in the DCP admin app.
+### Teams
+
+There are two main parts to the Chat Integration:
+
+- Adding the automation scripts & memory files to your Dataminer System.
+- Granting admin consent in the DCP admin app.
```
Comment (JordyGit) [inline README.md:28]: Move this line to Prerequisites > Teams and remove the rest from overview
Put the Table of contents under Overview subtitle

File: README.md [hunk 6]
```diff
@@ -1,56 +1,78 @@
...
 
-## Getting Started
+- The DataMiner System must be cloud-connected to the DataMiner Cloud Platform ([DCP](https://docs.dataminer.services/user-guide/Cloud_Platform/CloudAdminApp/Managing_cloud-connected_nodes.html)).
+- The CloudGateway [DxM](https://docs.dataminer.services/user-guide/Cloud_Platform/CloudAdminApp/Managing_cloud-connected_nodes.html) on your Dataminer System must be atleast version 2.9.0.
 
-### Limitations
+### Teams
 
-This chat integration is only supported by cloud-connected DataMiner systems.
+- The Dataminer App must be allowed in your [Teams](https://docs.microsoft.com/en-us/microsoftteams/manage-apps).
```
Comment (JordyGit) [inline README.md:39]: Everywhere MS Teams and not just Teams so it's aligned
```suggestion
- The Dataminer App must be allowed in your [Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/manage-apps).
```

File: README.md [hunk 7]
```diff
@@ -1,56 +1,78 @@
...
 ![Items](https://user-images.githubusercontent.com/109528797/186423143-bab59820-8e45-4315-a192-24b34398b502.png)
 
-* Connect your dataminer system to the cloud if it is not yet [Cloud Connected](https://docs.dataminer.services/user-guide/Cloud_Platform/AboutCloudPlatform/Connecting_your_DataMiner_System_to_the_cloud.html).
-
-* Grant access with [Admin Consent](https://docs.dataminer.services/user-guide/Cloud_Platform/CloudAdminApp/Granting_admin_consent.html).
+- Connect your DataMiner System to the cloud if it is not yet [Cloud Connected](https://docs.dataminer.services/user-guide/Cloud_Platform/AboutCloudPlatform/Connecting_your_DataMiner_System_to_the_cloud.html).
 
+- Grant access with [Admin Consent](https://docs.dataminer.services/user-guide/Cloud_Platform/CloudAdminApp/Granting_admin_consent.html).
 
+## Examples / Demo
```
Comment (JordyGit) [inline README.md:67]:
```suggestion
## Examples
```

File: README.md [hunk 8]
```diff
@@ -1,56 +1,78 @@
...
 
-### Usage
+### Teams
```
Comment (JordyGit) [inline README.md:69]:
```suggestion
### Microsoft Teams
```

File: README.md [hunk 9]
```diff
@@ -1,56 +1,78 @@
...
+- The CloudGateway [DxM](https://docs.dataminer.services/user-guide/Cloud_Platform/CloudAdminApp/Managing_cloud-connected_nodes.html) on your Dataminer System must be atleast version 2.9.0.
 
-### Limitations
+### Teams
 
-This chat integration is only supported by cloud-connected DataMiner systems.
+- The Dataminer App must be allowed in your [Teams](https://docs.microsoft.com/en-us/microsoftteams/manage-apps).
 
-### Prerequisites
+## Installation
```
Comment (JordyGit) [inline README.md:41]: I would remove Installation completely and put everything under Examples -> Teams -> Installation
And then below we could have Examples -> Teams -> Usage

**Review by JordyGit — CHANGES_REQUESTED** [review]

File: README.md (hunk 9 above)
Comment (RobbertVH) [reply README.md:41]: Examples
├── Teams
│   ├── Installation
│   └── Usage

like this

File: README.md [hunk 10]
```diff
@@ -1,55 +1,63 @@
...
+- [Deploy the DMAPP]() to your DataMiner System. Installing this package, will add the automation scripts for Microsoft Teams in the Automation module.
 
-* Connect your dataminer system to the cloud if it is not yet [Cloud Connected](https://docs.dataminer.services/user-guide/Cloud_Platform/AboutCloudPlatform/Connecting_your_DataMiner_System_to_the_cloud.html).
+- [Access the DataMiner System](https://docs.dataminer.services/user-guide/Getting_started/Accessing_DataMiner/Accessing_DataMiner.html) and open the Automation module (via apps > Automation). <details><summary>...</summary>
+![Gif-Automation](https://user-images.githubusercontent.com/109528797/186685478-9eac1cbf-f2d9-4c9a-8a6a-a2f499dbdcd9.gif)
 
-* Grant access with [Admin Consent](https://docs.dataminer.services/user-guide/Cloud_Platform/CloudAdminApp/Granting_admin_consent.html).
+</details>
 
+- Open the "memory files" tab and create memory files named *Teams, Channels and Chats*. The purpose of these files is to persist the created Teams, Channels and Chats.<details><summary>...</summary>
```
[the remark this answers is not in the document]
Comment (JordyGit) [reply README.md:43]: Maybe this would be more clear?

File: README.md [hunk 11]
```diff
@@ -1,55 +1,63 @@
...
+- Also create a memory file named YesOrNo with the following entries: *{postion:0, value:1, description:yes ; postion:1, value:0, description:no}*<details><summary>...</summary>
+![Gif-MemFilesYesorNo](https://user-images.githubusercontent.com/109528797/186685771-9d9c4155-1f58-4700-98aa-90ebd19c329e.gif)
 
-* Open the Automation module (via apps > Automation) and click on the "automation scripts" tab and double click the "Create Team" automation script. Fill in the fields for creating a new Team, **the fields are case sensitive.**
-* When pressing the "execute now" button, a Team is created in Microsoft Teams with the Dataminer bot.
+</details>
 
-<img src="https://user-images.githubusercontent.com/109528797/186440501-4850222e-c503-40fc-8f9a-003e8c7d5a30.png" width="684" height="757">
 
+#### Usage
```
Comment (JordyGit) [inline README.md:54]: Also for create channels, add member/owner and send channel notification ? or have only this one for create team and then do a more generic one like, 'repeat these steps for the other scripts'?

Maybe we have to explain you can send messages in a Channel or chat
A channel exists in a Team
A Chat is a group or private chat (no create support yet)

File: README.md [hunk 12]
```diff
@@ -60,17 +68,16 @@ Any advise for common problems or issues.
...
-* 0.2
-   * optimizations
-   * See [commit change]() or See [release history]()
-* 1.0
-   * Initial Release
+- 0.2
+  - Refactoring
+  - Generalizations
+- 1.0
+  - Initial Release
```
Comment (JordyGit) [inline README.md:79]: We shouldn't have these things here? we haven't released it yet so should be empty still
we don't want commit messages here?

**Review by JordyGit — CHANGES_REQUESTED** [review]

**Comment by JordyGit** [comment]
Looks good!

Only the DMAPP still needs to be filled in when it's ready :) I think we would just add the file in the repo or how could be provide a download url?

**Comment by RobbertVH** [comment]
> 

yes this can be done

**Pull request merged**